### PR TITLE
Fix update issue from debian version to official nginx version

### PR DESCRIPTION
**Pull request by YohanBrun**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM

File: tasks/setup-Debian.yml [hunk 1]
```diff
@@ -1,4 +1,16 @@
 ---
+- name: Ensure nginx-common package is absent
+  apt:
+    name: nginx-common
+    state: absent
```
Comment (gloutsch) [inline tasks/setup-Debian.yml:5]: why remove ?

File: tasks/setup-Debian.yml [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+  register: nginx_mask
+  become: yes
+
+- name: Ensure nginx service is unmasked
+  command: systemctl unmask nginx.service
```
Comment (gloutsch) [inline tasks/setup-Debian.yml:10]: http://docs.ansible.com/ansible/latest/systemd_module.html

**Comment by YohanBrun** [comment]
The latest version available on debian repository is 1.6.2. We needed a newer version, so we had to switch to the official nginx repository.
But there is a conflict between the 2 packages. The fact that nginx-common is present on the instance prevent us from upgrading the package.

The systemd module would have been nice, unfortunalty it is only available with ansible <2.2

**Pull request merged**